_component_design/radio-buttons.md

```diff
@@ -147,26 +186,98 @@ Form design best practices prohibit a radio button from existing in an error sta
 
 <section class="static-section" markdown="1">
 
-<div class="hxRow" markdown="1">
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Disabled
+
+A user can't select a radio button in the disabled state. In the disabled state, the radio button is locked and cannot be changed.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Disabled state"] [class:"image bg-light border"] %}
+<embed src="{{site.url}}/assets/images/components/inputs-and-controls/radio-buttons/radiobuttons-states-disabled.png" width="411"/>
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-#### Error state - expanded
+### Error
+
+An error state can apply to radio buttons when the user selects, deselects, and hovers over a checkbox.
```

Comment:
?? over a checkbox?  i'm confused here. do you mean radio button?

Reply:
Good catch! yes, that was a copy/paste error. Fixing!